databaker/bake.py

```diff
@@ -206,30 +216,33 @@ def value_for_dimension(dimension):
         values = {}
         values[OBS] = obj.value
 
-        for dimension in range(DATAMARKER, LAST_METADATA + 1):
+        for dimension in range(OBS+1, LAST_METADATA + 1):
             values[dimension] = value_for_dimension(dimension)
 
         # Mutate values
         # Special handling per dimension.
-
-        if not isinstance(values[OBS], float):  # NOTE xls specific!
-            ob_value, dm_value = parse_ob(ob)
-            values[OBS] = ob_value
-            # the observation is not actually a number
-            # store it as a datamarker and nuke the observation field
-            if values[DATAMARKER] == '':
-                values[DATAMARKER] = dm_value
-            elif dm_value:
-                logging.warn("datamarker lost: {} on {!r}".format(dm_value, ob))
-
-        if values[TIMEUNIT] == '' and values[TIME] != '':
-            # we've not actually been given a timeunit, but we have a time
-            # determine the timeunit from the time
-            values[TIMEUNIT] = datematch(values[TIME])
+        # NOTE  - varibales beinging SH_ ... are ependant on user choices from the bottom of the template file
+
+        if SH_Split_OBS:
+            if not isinstance(values[OBS], float):  # NOTE xls specific!
+                ob_value, dm_value = parse_ob(ob)
+                values[OBS] = ob_value
+                # the observation is not actually a number
+                # store it as a datamarker and nuke the observation field
+                if values[OBS+1] == '':
```

Comment:
Good point. How about we change structure_csv_default line 68 from a boolean? so..  
SH_Split_OBS = True

to somthing like:
SH_Split_OBS = DATAMARKER

the logic in bake.py could be set to ignore the special handling if SH_Split_OBS == None (so skip the logic if a user wants mixed numbers and data markings). If its not none the data markings would get dropped in the dimension/constant/column specified.

if values[OBS+1] == '':
becomes
if values[SH_Split_OBS] == '':

We could just write DATAMARKER straight back into bake.py but thats another constant the user will always have to include, trying to avoid those.